conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -217,12 +218,8 @@ object DynamicConformanceJob {
       case None    => withPartCols.count
       case Some(p) => p
     }
+    if (recordCount == 0) { handleEmptyOutputAfterConformance() }
```

Comment:
Tiny: I find the one-liner withing curly braces weird.
I would prefer the full:
```
if (recordCount == 0) {
  handleEmptyOutputAfterConformance()
}
```
or if to keep short because simple then
`if (recordCount == 0) handleEmptyOutputAfterConformance()`
The current state is a strange mixture, IMO 😉